vars/buildDockerImage.groovy

```diff
@@ -12,6 +12,12 @@ void call(Map args = [:]) {
     def build_qualifier = inputManifest.build.qualifier
     def build_number = args.buildNumber ?: "${BUILD_NUMBER}"
 
+    String image_tag =""
+
+    if (args.buildDate != null){
```

Comment:
Also add `!= 'null'` condition as sometimes jenkins does not do pure groovy and can cause weird behavior.